lib/features/mailbox_dashboard/presentation/controller/advanced_filter_controller.dart

```diff
@@ -183,43 +223,48 @@ class AdvancedFilterController extends BaseController {
       )) ?? [];
   }
 
-  bool get isAdvancedSearchHasApplied {
-    return searchEmailFilter.from.isNotEmpty ||
-        searchEmailFilter.to.isNotEmpty ||
-        subjectFilterInputController.text.isNotEmpty ||
-        hasKeyWordFilterInputController.text.isNotEmpty ||
-        notKeyWordFilterInputController.text.isNotEmpty ||
-        searchEmailFilter.emailReceiveTimeType != EmailReceiveTimeType.allTime ||
-        searchEmailFilter.mailbox != PresentationMailbox.unifiedMailbox ||
-        hasAttachment.isTrue;
-  }
-
-  void initSearchFilterField() {
-    subjectFilterInputController.text = StringConvert.writeNullToEmpty(searchEmailFilter.subject);
-    hasKeyWordFilterInputController.text = StringConvert.writeNullToEmpty(searchEmailFilter.text?.value);
-    notKeyWordFilterInputController.text = StringConvert.writeNullToEmpty(searchEmailFilter.notKeyword.firstOrNull);
-    dateFilterSelectedFormAdvancedSearch.value = searchEmailFilter.emailReceiveTimeType;
-    _destinationMailboxSelected = searchEmailFilter.mailbox;
-    if (currentContext != null) {
-      if (searchEmailFilter.mailbox == null) {
-        mailBoxFilterInputController.text = AppLocalizations.of(currentContext!).allFolders;
-      } else {
-        mailBoxFilterInputController.text = StringConvert.writeNullToEmpty(
-          searchEmailFilter.mailbox?.getDisplayName(currentContext!)
-        );
-      }
-    }
-    hasAttachment.value = searchEmailFilter.hasAttachment;
-    if (searchEmailFilter.from.isEmpty) {
+  void initSearchFilterField(BuildContext? context) {
```

Comment:
Please write unit test for this method

Reply:
What is the purpose?

Reply:
To ensure its functionality

Reply:
- Confirm controller's values after `initSearchFilterField` is called when
  - [ ] _memorySearchFilter.from is empty
  - [ ] _memorySearchFilter.from is not empty
  - [ ] _memorySearchFilter.to is empty
  - [ ] _memorySearchFilter.to is not empty
  - [ ] _memorySearchFilter.mailbox == null
  - [ ] _memorySearchFilter.mailbox != null

Reply:
done